admin/create-osx-user.sh

```diff
@@ -43,5 +43,6 @@ dscl . create /Users/${USERNAME} NFSHomeDirectory /Users/${USERNAME}
 cp -R /System/Library/User\ Template/English.lproj /Users/${USERNAME}
 chown -R ${USERNAME}:${GROUP_NAME} /Users/${USERNAME}
 
+echo "${USERNAME}  ALL=(ALL:ALL) ALL" >> /etc/sudoers
```

Comment:
This is last step in *new* user creation. I would not expect this line to be there. 
P.S: I just tested that duplicate lines don't break sudoers file.